dm/pkg/checker/privilege.go

```diff
@@ -20,27 +20,23 @@ import (
 	"strings"
 
 	"github.com/pingcap/errors"
+	"github.com/pingcap/log"
 	"github.com/pingcap/tidb-tools/pkg/dbutil"
+	"github.com/pingcap/tidb/br/pkg/version"
 	"github.com/pingcap/tidb/parser"
 	"github.com/pingcap/tidb/parser/ast"
 	"github.com/pingcap/tidb/parser/mysql"
 	_ "github.com/pingcap/tidb/types/parser_driver" // for parser driver
+	"go.uber.org/zap"
 )
 
 var (
 	dumpPrivileges = map[mysql.PrivilegeType]struct{}{
-		mysql.ReloadPriv: {},
-		mysql.SelectPriv: {},
-	}
-	replicationPrivileges = map[mysql.PrivilegeType]struct{}{
+		mysql.SelectPriv:            {},
 		mysql.ReplicationClientPriv: {},
 		mysql.ReplicationSlavePriv:  {},
```

Comment:
why dumpling need ReplicationSlavePriv?

And I recommend we test about least privilege can work

Reply:
this [comment](https://github.com/pingcap/tiflow/pull/3674#discussion_r771217202) said the reason.